adapter_pipelines/ss2_single_end/adapter.wdl

```diff
@@ -46,17 +46,6 @@ workflow AdapterSmartSeq2SingleCellUnpaired {
   String bundle_uuid
   String bundle_version
 
-  # fixed parameters
```

Comment:
Hmm, do we need to remove these in this PR? I thought this PR only cares about Optimus/Cellranger since the mouse SS2 is not ready?

Reply:
I wasn't aware that this effort was scoped to just Optimus/Cellranger... But it should be a harmless change that supports future addition of animal references (provided integration tests pass)